tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -1,19 +1,32 @@
 package e2e_test
 
 import (
+	"context"
 	"fmt"
 	"io"
+	"log"
 	"net/http"
 	"strings"
 	"time"
 
 	ginkgov2 "github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/openshift/oadp-operator/api/v1alpha1"
 	"github.com/openshift/oadp-operator/tests/e2e/lib"
 )
 
+var virtualMachineGVK = schema.GroupVersionResource{
```

Comment:
for standardization, would move this to `tests/e2e/lib/virt_helpers.go` or `tests/e2e/lib/virt_storage_helpers.go` and change name to GVR

Reply:
This was already in virt_helpers, but I decided to just make GetVmStatus public instead. That removed a chunk of redundant code and got rid of this extra definition.